snowfakery/data_generator_runtime_dom.py

```diff
@@ -42,6 +42,17 @@ def __init__(self):
     def render(self, context: RuntimeContext) -> FieldValue:
         pass
 
+    @contextmanager
+    def exception_handling(self, message: str, *args, **kwargs):
+        try:
+            yield
+        except DataGenError as e:
+            raise fix_exception(message, self, e)
+        except Exception as e:
+            message = message.format(*args, **kwargs)
+            print(f"{message} : {str(e)}", self.filename, self.line_num)
```

Comment:
We should use a logger, not print, so that it's possible to configure where the message goes.

Reply:
Sorry: It was a debugging print statement that slipped through. Almost want to add a lint rule for those...